src/react-imgix.jsx

```diff
@@ -386,7 +386,7 @@ class SourceImpl extends Component {
     if (disableSrcSet) {
       childProps[attributeConfig.srcSet] = src;
     } else {
-      childProps[attributeConfig.srcSet] = `${src}, ${srcSet}`;
```

Comment:
Interesting that this was in here as a direct rebuke to the helpful comment above it 🤔.